WordPress/src/main/java/org/wordpress/android/ui/main/ToolbarFragment.kt

```diff
@@ -0,0 +1,12 @@
+package org.wordpress.android.ui.main
+
+/**
+ * Interface for use with a fragment hosting a toolbar or actionbar.
+ */
+interface ToolbarFragment {
```

Comment:
Minor nit: can we rename this `MainToolbarFragment`?

Reply:
Also, 👍 for Kotlin.

Reply:
I was making this change and then it occurred to me that I had originally named this interface something similar but then renamed it to ToolbarFragment to keep it generic because technically there us nothing about it that ties it to main. It's just an interface that provides a contract for working with a fragment hosting a toolbar. We could just as easily use it in other areas of the app. Do you still want me to change it?

Reply:
I would prefer to change it, because at this point in time it's only used by the four main fragments hosted by the main activity. Although I'm sure I've violated this countless times, my preference is to name things to describe the current situation. If the situation changes, renaming is simple enough.

Reply:
Fair point. Fixed in 6cc403c2904ce0c36421e6353016b82e84fe7f33